var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -126,6 +127,7 @@ class Openblas(MakefilePackage):
     conflicts('+consistent_fpcsr', when='threads=none',
               msg='FPCSR consistency only applies to multithreading')
 
+    conflicts('threads=pthreads', when='~locking', msg='Pthread support requires +locking')
```

Comment:
Shouldn't we add also:
```python
conflicts('threads=openmp', when='~locking', msg='OpenMP support requires +locking')
```

Reply:
I thought so for a moment, but the OpenBLAS docs read:
```
This is done automatically for USE_THREAD=1 , and should not be necessary when USE_OPENMP=1.
```
(Please see PR description :) )

Reply:
Sure, not talking about the recipe, just the spec. I think `openblas threads=openmp ~locking` won't make sense (but we can work on that in a later PR).

Reply:
I think if I follow the docs that this is advertised as working, but I would not default to it reading the wording :D